packages/preview/splash-0.3.0/README.md

```diff
@@ -0,0 +1,15 @@
+# typst-palettes
+
+A library of color palettes for Typst.
+
+This library provides different color palettes with human-readable names in
+Typst dictionaries. Currently there are just a few different palettes to choose
+from. Any contributions or suggestions are welcome!
```

Comment:
Ideally the README would also state that you should look into the Typst files for the palette and color names or maybe list the available palettes here.

Reply:
There is a generated document with all the colors but I don't want to include it in the distributed files due to the unnecessary added size. I hope this is sufficient.

Reply:
Ah that's great. Then, I would propose linking "generated documentation" directly to that PDF.